Meadow.CSharp.Template/MeadowApp.cs

```diff
@@ -59,10 +65,9 @@ void ShowColorPulse(Color color, TimeSpan duration)
 
 		void ShowColor(Color color, TimeSpan duration)
 		{
-			Console.WriteLine($"Color: {color}");
```

Comment:
The rest of this class looks like lifecycle updates. Not opposed here, but was this removal intentional? It is still present in the F# and VB versions.

Reply:
yeah did a little cleanup and consistency while I was in there

Reply:
Okay...I see it now. The other cycle colors logging is there, but this method isn't used at all here. Should we just delete it all?